lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -62,8 +62,8 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking site_configuration_values.HOMEPAGE_URL %}"><img
-                                    src="https://mitxonline.mit.edu/static/images/mitx-online-logo.png" width="auto"
-                                    height="40" alt="{% filter force_escape %}{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}{% endfilter %}"/></a>
+                                    src="{{ logo_url }}" width="125"
+                                    height="35" alt="{% filter force_escape %}{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}{% endfilter %}"/></a>
```

Comment:
125x35 was a suitable lower size for xPRO logo based on It's original size. Is it suitable for MITx Online logo as well ? The dimensions I see from the [Figma link](https://www.figma.com/design/GFl0GZREDaWmdRytsNiET2/MIT-Logo-Header-%2F-Footer?node-id=1-2&t=FaHM0xe4CHQVWD6X-0) are 121.55x33.

Reply:
It seems to me 124.3x33px as per [figma](https://www.figma.com/design/GFl0GZREDaWmdRytsNiET2/MIT-Logo-Header-%2F-Footer?node-id=1-2&t=FaHM0xe4CHQVWD6X-0). So I set this size for now. Will update if this needs adjustment.